File: dotnet/Atlas.Averbacao/Api/Saude/SaudeController.cs
using Atlas.Averbacao.Api.Saude;
using Microsoft.AspNetCore.Mvc;

namespace Atlas.Averbacao.Api.Saude;

[ApiController]
[Route("v1/saude")]
public sealed class SaudeController : ControllerBase
{
    /// <summary>Saúde do processo — sem autenticação.</summary>
    [HttpGet]
    [ProducesResponseType(typeof(SaudeResponse), 200)]
    public SaudeResponse Get() => new();
}

File: java/src/main/java/br/com/atlas/averbacao/api/saude/SaudeController.java
package br.com.atlas.averbacao.api.saude;

import io.swagger.v3.oas.annotations.Operation;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SaudeController {
    @Operation(summary = "Saúde do processo — sem autenticação")
    @GetMapping("/v1/saude")
    public SaudeResponse get() {
        return new SaudeResponse();
    }
}
